tests/acceptance/features/coreApiSharePublicLink1/changingPublicLinkShare.feature

```diff
@@ -1,6 +1,9 @@
 @api @issue-1276 @issue-1277 @issue-2079 @issue-1269
 
 Feature: changing a public link share
+  As a user
+  I want to set permission in the public link share
+  So that public can only have designated authorization
```

Comment:
```suggestion
  As a user
  I want to set the permissions of a public link share
  So that people who have the public link only have the designated authorization
```